src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/EventLogInternal.cs

```diff
@@ -1250,7 +1250,7 @@ private void VerifyAndCreateSource(string sourceName, string currentMachineName)
             if (!EventLog.SourceExists(sourceName, currentMachineName, true))
             {
                 Mutex mutex = null;
-                RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.